ocaml/xapi/xapi_periodic_scheduler.ml

```diff
@@ -88,7 +88,7 @@ let loop () =
       else
         let next = with_lock lock (fun () -> Ipq.maximum queue) in
         let now = Mtime_clock.now () in
-        if next.Ipq.time < now then (
+        if Mtime.compare next.Ipq.time now < 0 then (
```

Comment:
```suggestion
        if Mtime.is_earlier next.Ipq.time ~than time then (
```

Reply:
almost, but had to use `now` instead of `time`, I pushed that as a separate commit.